### ISSUE TITLE
Dev: fix Alert: Layout table on Archived Projects page

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/tdm-calculator#3042]
### Overview
We need to fix the Alert(s): Layout table on Archived Projects so that page can be WCAG compliant.

#### Details
Category: Alerts
Name: Layout table
(The Accessibility wiki for this Alert has no solution yet)

Page that will be worked on:
Archived Projects

### Action Items
- [x] Review wiki page: Accessibility, How to run the WAVE browser extension (Resource 1.01)
   - [x] Install the WAVE browser extension
- [x] Log in as the user indicated (resource 1.02.01) 
- [x] navigate to the page you are going to fix (resource 1.02.02)
- [x] Open the TDM WAVE Accessibility Audit Spreadsheet, for this Alert (resource 1.03) and find the row that relates to this  issues page.
- [x] Run the WAVE extension with no elements focused, or clicked (resource 1.04)
- [x] Find the Alert shown in the Spreadsheet (1.03)
- [x] This Alert has been resolved on other pages, so look at the other issues where this was solved (resource 1.05.01 see Project Team Alert guidance section)
- [x] Apply solution
- [x] write and add the appropriate details to the `Project Team Solution` section (resource 1.05.01.01)
- [x] Submit PR (include link to `Project Team Solution` section (resource 1.05.01.01)
- [x] Add PR to spreadsheet in PR column next to Alert (resource 1.03)
- [x] On this issue
   - [x] Add labels: 
      - `ready for accessibility lead`
      - `ready for product`
   - [x] Change status to: `Technical Review`
- [x] Get sign-off on WIKI page by
   - [x] Accessibility lead
   - [x] Product lead
- [ ] release the dependencies on #2893
- on the following issues
   - issue #3043
   - issue #3044
      - [ ] release the dependency
      - [ ] add the `ready for accessibility lead` label
      - [ ] move the issue to the new issue approval column

### Resources/Instructions
- 1.01[How to run the WAVE browser extension](https://hackforla.github.io/accessibility/getting-started/wave-browser-extension/)
- 1.02 Application details
   - 1.02.01 User Profile Required: security admin
   - 1.02.02 Page URL: https://tdm-dev.azurewebsites.net/archivedprojects
      - Requires sign in: TRUE
      - Can access page directly from URL: FALSE
- 1.03 [TDM WAVE Accessibility Audit Spreadsheet, tab: 2. Audit Log, filter: Alert: Layout table](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=0#gid=0&fvid=403758315)
   -  Spreadsheet Row(s): 43
- 1.04 https://hackforla.github.io/accessibility/getting-started/perform-wave-audit
- 1.05 [Accessibility Known Issues Index](https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/)
    - 1.05.01 [WAVE Known Accessibility Issue: Alert: Layout Table](https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/alert-layout-table/)
       - 1.05.01.01 [WAVE Known Accessibility Issue: Alert: Layout Table, section Project Team Solution](https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/alert-layout-table/#project-team-solution)

Additional reading (optional)
- TDM [Accessibility Audits](https://hackforla.github.io/accessibility/)

#### Slides
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
   - ~[Staging Deck, slide 3042](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3da5aa8ec92_6_0#slide=id.g3da5aa8ec92_6_0)~
- Release deck
   - [2026-MM-DD Release deck, slide 3042]

**Comment by Rabia2219** [comment]
@geolunalg This issue requires the developer to create a solution, apply it to the issue and write the solution into the WIKI page on the accessibility repo.

Please add a level label. You can choose from one of the following:

- ~`good first issue`~
- `level: easy`
- `level: medium`
- `level: hard`

Please add a size label. You can choose from one of the following:

| label        | Description                             |
| ------------ | --------------------------------------- |
| ~size: 0.25pt~ |~Can be done in 1.5 hours or less~        |
| ~size: 0.5pt~  | ~Can be done in 2-3 hours or less~        |
| ~size: 1pt~    | ~Can be done in 4-6 hours~              |
| size: 2pt    | Can be done in 7-12 hours               |
| size: 3pt    | Can be done in 13-18 hours              |
| size: 5pt    | Can be done in 19-30 hours              |
| size: 8pt    | Can be done in 31-48 hours              |
| size: 13+pt  | Must be broken down into smaller issues |


Once you have added the level and size labels, please:
- remove the `level: missing` label
- remove the `size: missing` label
- remove the `ready for accessibility lead` label
- add the `ready for prioritization` label

**Comment by Philc90** [comment]
Verified this WAVE Alert no longer appears on the Archived Projects page, appears to have been fixed in a previous run.

<img width="1849" height="801" alt="Image" src="https://github.com/user-attachments/assets/c4d39f47-b5e1-4b19-9d92-289fbfb72998" />

**Comment by Philc90** [comment]
Determined that the Layout Table Alert was fixed in PR #3068 for an unrelated (non-Accessibility) ticket that happened to also fix the WAVE alert.

The root cause for the Layout Table alert was that the table header row was using td tags instead of th tags.

This was fixed in PR #3068 for ticket #2837 which was for fixing visual styles

**Comment by Philc90** [comment]
Opened PR in Accessibility to document solution for Layout Table Alert: https://github.com/hackforla/accessibility/pull/59

**Comment by ExperimentsInHonesty** [comment]
@Philc90 
- Please review https://github.com/hackforla/accessibility/pull/59#discussion_r3133433734

**Comment by Philc90** [comment]
Progress: Completed pending PR review for https://github.com/hackforla/accessibility/pull/59
Blockers: None
Availability: before 5/6
ETA: end of week

**Comment by Philc90** [comment]
> [@Philc90](https://github.com/Philc90)
> 
>     * Please review [update Alert - Layout table page accessibility#59 (comment)](https://github.com/hackforla/accessibility/pull/59#discussion_r3133433734)

@ExperimentsInHonesty could you please re-review the PR from accessibility wiki? https://github.com/hackforla/accessibility/pull/59
I updated in this commit to copy the exact HTML that triggers the WAVE error:
https://github.com/hackforla/accessibility/pull/59/changes/26ac3af12caeac3a2285946b0df3f09f5664f523

**Comment by Philc90** [comment]
Progress: Completed - no PR needed in tdm-calculator, PR for Accessibility wiki page for Layout Table merged (https://github.com/hackforla/accessibility/pull/59)
Blockers: None
Availability: Rest of week
ETA: Completed

**Comment by Philc90** [comment]
I forgot to add this screenshot earlier:

Verified this Alert no longer appears on the Archived Projects page, appears to have been fixed already.

<img width="1572" height="873" alt="Image" src="https://github.com/user-attachments/assets/de4585e3-041a-4c50-b395-e681590a6d5b" />

However, the accessibility wiki page for documenting solutions for Alert - Layout Table was not complete. This was updated and merged in PR: https://github.com/hackforla/accessibility/pull/59
